assets/app/widgets/crowbar-circle/crowbar-circle.directive.js

```diff
@@ -0,0 +1,21 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('crowbarWidgets')
+        .directive('crowbarCircle', crowbarCircle);
+
+    function crowbarCircle() {
+        return {
+            restrict: 'E',
+            scope: {
+                active: '='
+            },
+            link: function (scope, element) {
+                scope.$watch('active', function(newVal) {
```

Comment:
This is the main functionality of this directive. Monitor `active` scope attribute and set CSS classes on DOM element appropriately.